python/doc/usecases/use_case_fireSatellite.rst

```diff
@@ -0,0 +1,272 @@
+.. _use-case-fireSatellite:
+
+The Fire Satellite model
+========================
+
+The fire satellite model is a multidisciplinary test case that involves 3 disciplines: Power, Orbit and Attitude \& control. This usecase has been firstly proposed by Wertz et al. The model deals with the design of a realistic satellite, the goal of which is to detect and monitor forest fires from Earth's orbit, through the use of optical sensors. Three disciplines are needed to design this satellite. The orbit discipline is responsible of computing the orbit period, the satellite velocity, the maximal slewing angle and the eclipse period. The power discipline is used to estimate the total power of the system and the area of the solar arrays. Finally, the attitude \& control discipline computes the total torque of the satellite and the power required for the attitude control system.
+These disciplines exchange several coupling variables. The multidisciplinary analysis that is used to compute the converged values of the coupling variables is performed through a Fixed Point Iteration algorithm.

```

Comment:
better avoid these very long lines

Reply:
I don't understand. Do you propose to remove this description?

Reply:
I believe Julien means that lines 6 and 7 should not each contain a whole paragraph. For example, the content of line 7 could be split between 3 lines like this:
```suggestion
These disciplines exchange several coupling variables.
The multidisciplinary analysis that is used to compute the converged values
of the coupling variables is performed through a Fixed Point Iteration algorithm.
```
Keep in mind the file is not only source code for an HTML page, but also a Python script that can be downloaded and edited by users. Splitting these lines will not change anything to the HTML rendering, but will make the Python file easier to read in text editors.

Having short lines also makes it easier to track changes through Git (because it highlights lines that change).

Reply:
Ok, thanks!